### PR TITLE
[SPARK-8660][ML] Convert JavaDoc style comments inLogisticRegressionSuite.scala to regular multiline comments, to make copy-pasting R commands easier

### PR DESCRIPTION
**Pull request by Rosstin**
Converted JavaDoc style comments in mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala to regular multiline comments, to make copy-pasting R commands easier.

### CODE REVIEW

**Comment by rxin** [comment]
Jenkins, test this please.

**Comment by rxin** [comment]
@Rosstin can you update the pull request title to make it more descriptive?

**Comment by Rosstin** [comment]
@rxin done, hope that's not too long

**Comment by rxin** [comment]
Thanks - one more thing. Can you change Spark-8660 to [SPARK-8660] ?

**Comment by Rosstin** [comment]
@rxin Of course, my bad

**Comment by rxin** [comment]
Thanks - LGTM. We can merge this once Jenkins comes back happy.

**Comment by rxin** [comment]
Thanks - merging this in master.

File: mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala [hunk 1]
```diff
@@ -211,22 +211,22 @@ class LogisticRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
...
-     *                     s0
-     * (Intercept)  2.8366423
-     * data.V2     -0.5895848
-     * data.V3      0.8931147
-     * data.V4     -0.3925051
-     * data.V5     -0.7996864
+    /*
+       Using the following R code to load the data and train the model using glmnet package.
+
+       > library("glmnet")
```
Comment (mengxr) [inline mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala]: @Rosstin Could you also remove `>`s? Then people can easily copy & paste.

File: mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala [hunk 2]
```diff
@@ -211,22 +211,22 @@ class LogisticRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
...
+       > data <- read.csv("path", header=FALSE)
+       > label = factor(data$V1)
+       > features = as.matrix(data.frame(data$V2, data$V3, data$V4, data$V5))
+       > weights = coef(glmnet(features,label, family="binomial", alpha = 0, lambda = 0))
+       > weights
```
Comment (mengxr) [inline mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala]: If we remove `>`, please insert an empty line to separate commands and results.

File: mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala (hunk 1 above)
Comment (Rosstin) [inline mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala]: Sure, no problem.

**Pull request closed without merging**